suppliers/src/UI/embc-supplier/src/app/app.module.ts

```diff
@@ -19,7 +20,13 @@ import { CoreModule } from './core/core.module';
     AppRoutingModule,
     LayoutModule,
     HttpClientModule,
-    ReactiveFormsModule
+    ReactiveFormsModule,
+    OAuthModule.forRoot({
+      resourceServer: {
+        customUrlValidation: url => url.startsWith('/api') && !url.endsWith('/configuration'),
+        sendAccessToken: true
```

Comment:
This actually replaces the need to have the authguard, the library should send the token automatically